src/test/java/org/cactoos/scalar/AndInThreadsTest.java

```diff
@@ -109,7 +108,7 @@ void iteratesList() {
             "Must iterate a list with a procedure",
```

Comment:
@victornoel I see we iterate here with a function. I think the comment should be `Must iterate a list with a function`